build/scripts/install_python_modules.sh

```diff
@@ -156,7 +163,12 @@ source "${OUTPUTDIR}/bin/activate"
 # virtual environment path.
 
 progress "PYTHON" "pip install ${requirement_paths}"
-${OUTPUTDIR}/bin/python -m pip -q install --upgrade ${requirement_options}
+
+if [ ! -z "${local_packages_dir}" ]; then
```

Comment:
Please simplify this by just accumulating or setting the options with which to invoke python and then just invoke python outside the conditional.

Reply:
Will do.